graphql/resolvers.go

```diff
@@ -770,20 +770,30 @@ func (r *queryResolver) PatchBuildVariants(ctx context.Context, patchID string)
 	if err != nil {
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting tasks for patch `%s`: %s", patchID, err))
 	}
+	variantDisplayName := make(map[string]string)
 	for _, task := range tasks {
 		t := PatchBuildVariantTask{
 			ID:     task.Id,
 			Name:   task.DisplayName,
-			Status: task.GetDisplayStatus(),
+			Status: task.Status,
```

Comment:
why do we use `task.Status` instead of `task.GetDisplayStatus()`?

Reply:
Great Catch! I must have accidentally changed it while trying to figure this out.